core/debug/loading-inspection-manager.php

```diff
@@ -0,0 +1,54 @@
+<?php
+namespace Elementor\Core\Debug;
+
+use Elementor\Core\Debug\Classes\Inspection_Base;
+use Elementor\Core\Debug\Classes\Theme_Missing;
+use Elementor\Core\Debug\Classes\Htaccess;
+
+class Loading_Inspection_Manager {
+
+	public static $_instance = null;
+
+	public static function instance() {
+		if ( null === self::$_instance ) {
+			self::$_instance = new Loading_Inspection_Manager();
+		}
+		return self::$_instance;
+	}
+
+	/** @var Inspection_Base[] */
+	private $tests = [];
+
+	public function register_tests() {
```

Comment:
Rename `test` to `inspection` here and future.

Reply:
done